### DISCUSSION TITLE
Porting guide

### DISCUSSION THREAD

**Discussion by tomstorey** [EtchedPixels/FUZIX#1055] [Q&A]
Maybe Ive missed something obvious, but is there a guide anywhere that details the process for writing a port for a new platform?

I am working on a 68000 based computer, and would like to get FUZIX running on it as a more capable system than CP/M-68k, but Im a bit stumped about where to start.

Probably I can just take one of the existing 68k ports and start fumbling around and modify it to suit my system, but I wondered if there was some kind of checklist of things to do?

Thanks

**Comment by EtchedPixels** [comment]
For 68000 I'd clone an existing target and adjust the details.

There are some notes in 68000.md but if it's a flat 68K then usually you just to plug new numbers and driver details into something like 68knano and go from there.

**Reply by DoctorWkt** [reply]
Tom, I just completed a port to an m68k SBC, the Rosco r2. Maybe looking at my port might be useful: https://github.com/DoctorWkt/FUZIX/tree/platform-rosco-r2/Kernel/platform/platform-rosco-r2

**Reply by tomstorey** [reply]
Thanks!

I'll have to have a look in to this at some stage. Feeling a bit snowed under with a load of projects that I want to work on, but I'll get there eventually.

Need to cull my list and focus on one or two things I think. :)
